pkg/apis/pipeline/v1alpha1/pipeline_types.go

```diff
@@ -223,7 +223,19 @@ type PipelineTaskParam = v1beta1.PipelineTaskParam
 
 // PipelineTaskCondition allows a PipelineTask to declare a Condition to be evaluated before
 // the Task is run.
-type PipelineTaskCondition = v1beta1.PipelineTaskCondition
+type PipelineTaskCondition struct {
```

Comment:
I'm not sure it makes sense to keep this around, since there won't be any code to handle conditions in v1alpha1 pipelines anyway